page_test.go

```diff
@@ -647,6 +648,13 @@ func TestScreenshotFullPage(t *testing.T) {
 		g.mc.stubErr(1, proto.EmulationSetDeviceMetricsOverride{})
 		p.MustScreenshotFullPage()
 	})
+
+	g.Panic(func() {
+		g.mc.stub(1, proto.PageGetLayoutMetrics{}, func(send StubSend) (gson.JSON, error) {
+			return gson.New(proto.PageGetLayoutMetricsResult{}), nil
+		})
+		p.MustScreenshotFullPage()
```

Comment:
Should use `p.ScreenshotFullPage`

Reply:
thank you for the review. Unfortunately, there are two problems with this suggestion:
1) `p.ScreenshotFullPage undefined (type *rod.Page has no field or method ScreenshotFullPage)` so maybe you meant `p.Screenshot(true, nil)`?
2) If I don't use the Must method, the `panic` won't be triggered hence the `g.Panic()` I'm wrapping this call in.